views/home.ejs

```diff
@@ -84,6 +84,7 @@
     <form method="POST" action="/pricelist" id="frm" style="padding-left: 20px;">
         <button class="btn btn-danger" id="removeBtn" type="submit" form="frm" value="Submit">Delete selected</button>
         <button type="button" class="btn btn-danger" data-toggle="modal" data-target="#areYouSure">Delete all items</button>
+        <p><%= pricelist.length %> Item(s)</p>
```

Comment:
Add something so its not `Item(s)` but item / items depending on the length being 1 or not

Reply:
Second thought add it to the button instead, `Delete all x items`